src/js/jsx/PanelSet.jsx

```diff
@@ -269,11 +276,19 @@ define(function (require, exports, module) {
             this.getFlux().actions.preferences.setPreferences(nextState);
             headlights.logEvent("user-interface", "panel-toggle", panelName);
         },
+        
+        /**
+         * True if the current docuemnt is initialized and its panels should be displayed.
+         * 
+         * @return {Boolean}
+         */
+        _hasActiveDocument: function () {
+            return this.state.activeDocument && this.state.activeDocumentInitialized && this.state.documentIDs.size > 0;
+        },
 
         render: function () {
-            var documentIDs = this.state.documentIDs,
-                activeDocument = this.state.activeDocument,
-                hasActiveDocument = activeDocument && this.state.activeDocumentInitialized && documentIDs.size > 0;
+            var activeDocument = this.state.activeDocument,
```

Comment:
Might be slightly nicer to just have `_getActiveDocument` instead of `_hasActiveDocument` which returns the document if the other conditions hold and null otherwise?